jerry-core/ecma/operations/ecma-jobqueue.c

```diff
@@ -28,26 +34,221 @@
  */
 
 /**
+ * Create a PromiseReactionJob
+ *
+ * @return pointer to the PromiseReactionJob
+ */
+static ecma_job_promise_reaction_t *
+create_promise_reaction_job (ecma_value_t reaction, /**< PromiseReaction */
```

Comment:
These functions do not have ecma_ or ecma_job_ prefix. Even if they are static functions.